core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/annotations/CandidateForRemoval.kt

```diff
@@ -0,0 +1,9 @@
+package org.jetbrains.kotlinx.dataframe.annotations
+
+/**
+ * Functions that can be replaced with existing API such as shortcuts without clean value,
+ * or something not properly designed. Can be considered to be removed from API.
```

Comment:
I think this should be one sentence, no?

Reply:
Rephrased it a little